src/quacc/recipes/torchsim/core.py

```diff
@@ -0,0 +1,336 @@
+"""Core recipes for TorchSim."""
+
+from __future__ import annotations
+
+from importlib.util import find_spec
+from typing import TYPE_CHECKING, Any
+
+from monty.dev import requires
+
+from quacc import __version__, job
+from quacc.recipes.torchsim._base import (
+    pick_model,
+    process_binning_autobatcher_dict,
+    process_in_flight_autobatcher_dict,
+    process_trajectory_reporter_dict,
+)
+from quacc.runners._base import BaseRunner
+from quacc.schemas.torchsim import CONVERGENCE_FN_REGISTRY, ConvergenceFn, TSModelType
+
+has_torchsim = bool(find_spec("torch_sim"))
+if has_torchsim:
+    import torch_sim as ts
```

Comment:
I agree with Aaron's comment on the atomate2 PR that it'd be nice not to call this `ts`. Leaving it as `torch_sim` seems fine.